components/ww3/cime_config/buildnml

```diff
@@ -50,8 +51,12 @@ def buildnml(case, caseroot, compname):
     # Verify wav grid is supported
     #--------------------------------------------------------------------
 
-    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30", "wQU100", "wQU225EC30to60E2r2")
-    expect(wav_grid in wav_grid_supported, "WAV_GRID '{}' is not supported in ww3. Choose from: '{}'".format(wav_grid,' ,'.join(wav_grid_supported)))
+    wav_grid_supported = ("null",
+                          "wQU225EC60to30sp50x36",
+                          "wQU225EC30to60E2r2sp50x36",
+                          "wQU225EC30to60E2r2sp36x36",
+                          "wQU225EC30to60E2r2sp25x36")
+    expect((wav_grid+wav_spec)) in wav_grid_supported, "Combination of WAV_GRID {} and WAV_SPEC {} is not supported in ww3. Choose from: '{}'".format(wav_grid,wav_spec,wav_grid_supported) )
```

Comment:
```suggestion
    expect((wav_grid+wav_spec) in wav_grid_supported, "Combination of WAV_GRID {} and WAV_SPEC {} is not supported in ww3. Choose from: '{}'".format(wav_grid,wav_spec,wav_grid_supported) )
```

There's an extra parenthesis here that I noticed causes an error in `./case.setup`

Reply:
great catch - thank you!